nemo/collections/multimodal/modules/stable_diffusion/attention.py

```diff
@@ -293,10 +311,20 @@ def __init__(
         self.use_flash_attention = use_flash_attention
 
         if dim_head <= 160 and (dim_head % 8) == 0 and flash_attn_installed:
```

Comment:
Is flash_attn_installed also a requirement for TE attention? Otherwise we might need to rearrange the if-else logic here.